src/focus-trap-react.js

```diff
@@ -48,17 +53,82 @@ class FocusTrap extends React.Component {
     this.updatePreviousElement();
   }
 
+  // TODO: Need more test coverage for this function
+  getNodeForOption(optionName) {
+    const config = {
+      returnFocusOnDeactivate: true,
+      escapeDeactivates: true,
+      delayInitialFocus: true,
```

Comment:
Why are these here? None of these are options that are typed as `HTMLElement | SVGElement | string | () => HTMLElement | SVGElement` which is what we're interested in here.

Reply:
I just grabbed that straight from core `focus-trap`.

I didn't look much into it. I just assumed that they were necessary since it was in core.

https://github.com/focus-trap/focus-trap/blob/81898771081e524847c8c4dc3a942cc5f6340378/index.js#L90

I see that it is used for setting up default settings in core focus-trap. Not necessary here.

It's late for me. I'll need to fix it up tomorrow.